### PR TITLE
Add detailed erc20

### PR DESCRIPTION
**Pull request by GregTheGreek**
This PR changes the behaviour of the erc20 contact. 

Previous erc20 contracts were deployable with the `name` and `symbol` parameter. This PR extends the previous contract to allow a third argument `decimal`.

### CODE REVIEW

File: cb-sol-cli/cmd/deploy.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const deployCmd = new Command("deploy")
     .option('--erc20', 'Deploy erc20 contract')
     .option('--erc20Symbol <symbol>', 'Name for the erc20 contract', "")
     .option('--erc20Name <name>', 'Symbol for the erc20 contract', "")
+    .option('--erc20Decimal <amoutn>', 'Decimal for erc20 contract', 18)
```
Comment (lastperson) [inline cb-sol-cli/cmd/deploy.js:21]:
```suggestion
    .option('--erc20Decimals <amount>', 'Decimals for erc20 contract', 18)
```

File: cb-sol-cli/cmd/deploy.js [hunk 2]
```diff
@@ -158,7 +159,7 @@ async function deployBridgeContract(args) {
 
 async function deployERC20(args) {
     const factory = new ethers.ContractFactory(constants.ContractABIs.Erc20Mintable.abi, constants.ContractABIs.Erc20Mintable.bytecode, args.wallet);
-    const contract = await factory.deploy(args.erc20Name, args.erc20Symbol, { gasPrice: args.gasPrice, gasLimit: args.gasLimit});
+    const contract = await factory.deploy(args.erc20Name, args.erc20Symbol, args.erc20Decimal, { gasPrice: args.gasPrice, gasLimit: args.gasLimit});
```
Comment (lastperson) [inline cb-sol-cli/cmd/deploy.js:162]:
```suggestion
    const contract = await factory.deploy(args.erc20Name, args.erc20Symbol, args.erc20Decimals, { gasPrice: args.gasPrice, gasLimit: args.gasLimit});
```

**Review by lastperson — CHANGES_REQUESTED** [review]
Typos. Otherwise I like it!

**Comment by ansermino** [comment]
We already get a lot of questions about where the ERC20 contract comes from (since it's currently imported from NPM). We should include source here not just ABI.

**Comment by GregTheGreek** [comment]
@ansermino updated.

**Review by vezenovm — DISMISSED** [review]
Tests passing now

**Review by lastperson — APPROVED** [review]

**Pull request merged**